lib/ffmpeg/transcoderbase.py

```diff
@@ -210,6 +211,18 @@ def transcode(self):
           iopts.format(osencoded), oopts.format(vppscale, self.frames, osyuv))
         self.check_resolution(output, osencoded)
         self.check_metrics(yuv, refctx = [(n, channel)])
+
+        is_hdr = vars(self).get("HDR", 0)
```

Comment:
Should there be a per-output option?  Maybe some case can specify a codec that is unable to transcode HDR meta.

Reply:
also, please use lowercase for variable name: `HDR` -> `hdr`